README.md

```diff
@@ -20,8 +20,11 @@ purchasing and full length streaming. More details at
 
 
 A serverside javascript client for the 7digital API .
-Full code documentation for the most recent release can be found [here](http://raoulmillais.github.com/node-7digital-api/api.html)
+Full code documentation for the most recent release can be found [here](http://raoulmillais.github.com/node-7digital-api/api.html).
 
+To map documentation endpoints with this library methods, best is to look
+ at the [API definition file](https://github.com/7digital/7digital-api/blob/master/assets/7digital-api-schema.json).
+ 
```

Comment:
I'd replace 
"To map documentation endpoints with this library methods, best is to look"
with
"To map documentation endpoints with this library's methods, it's best to look at the" 

reads a little better ;-)